hvplot/__init__.py

```diff
@@ -25,17 +25,17 @@
 >>> idx = pd.date_range('1/1/2000', periods=1000)
 >>> df  = pd.DataFrame(np.random.randn(1000, 4), index=idx, columns=list('ABCD')).cumsum()
 
-Import the hvplot extension for your data source
+Import the hvplot extension for your data source and optionally set the plotting backend
 
 >>> import hvplot.pandas
+>>> # import hvplot
```

Comment:
The import is actually not required, `import hvplot.pandas` does import `hvplot` in the namespace.

```suggestion
```